paws/values.yaml

```diff
@@ -20,126 +66,172 @@ jupyterhub:
       users: 
         - Yuvipanda
         - BDavis_(WMF)
+        - BStorm_(WMF)
         - Andrewbogott
```

Comment:
Adding Arturo, Hieu, and Nicholas here might not be a bad idea.

Reply:
This is true. I can add a commit for it. They can also be easily added to the running hub, but this would preserve that setting when people aren't using mysql mode.

Reply:
Done!